http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/VPCHelper.scala

```diff
@@ -55,13 +55,20 @@ class VPCHelper(config: VPCHelperConfig,
       )
       .build
 
-  private def networkUri(googleProject: GoogleProject, vpcConfig: VPCConfig): String =
-    s"projects/${googleProject.value}/global/networks/${vpcConfig.value}"
+  def buildNetworkUri(googleProject: GoogleProject, vpcConfig: VPCConfig): String =
```

Comment:
```suggestion
  private def buildNetworkUri(googleProject: GoogleProject, vpcConfig: VPCConfig): String =
```

Reply:
I would do `private[util]` and test the logic in unit test